ui/shared/utils/constants.js

```diff
@@ -590,6 +591,66 @@ const ORDERED_VEP_CONSEQUENCES = [
     value: 'INV',
     group: VEP_GROUP_SV,
   },
+  {
+    description: 'A loss of function effect',
+    text: 'Loss of function',
+    value: 'LOF',
+    group: VEP_GROUP_SV_CONSEQUENCES,
+  },
+  {
+    description: 'A loss of function effect via intragenic exonic duplication',
+    text: 'Loss Of Functionn Duplication',
+    value: 'DUP_LOF',
+    group: VEP_GROUP_SV_CONSEQUENCES,
+  },
+  {
+    description: 'A copy-gain effect',
+    text: 'Copy Gain',
+    value: 'COPY_GAIN',
+    group: VEP_GROUP_SV_CONSEQUENCES,
+  },
+  {
+    description: 'Having genes partially overlapped by the SV\'s duplication',
```

Comment:
This is grammatically inconsistent with the other descriptions. Lets do `A duplication partially overlapping the gene`